lib/mods/replace-methods.js

```diff
@@ -38,26 +38,27 @@ function replaceGlobalMethodCalls(node, imports) {
   if (nodeTypes.isPropertyAccessExpression(node) &&
       nodeTypes.isCallExpression(node.getParent()) &&
       isGlobal(node.getExpression())) {
-    const baseType = node.getExpression().getType();
-    let callerType = baseType.getSymbol().getName();
+    let baseType = node.getExpression().getType();
```

Comment:
This still isn't really what I'm thinking. Now baseType is used for two different purposes: sometimes it's a type, and sometimes it's a name.

Here is my suggestion:

```js
let baseType = node.getExpression().getType();
let baseConstructSignature = baseType.getConstructSignatures()[0];

if (baseConstructSignature) {
  baseType = baseConstructSignature.getReturnType();
}

const baseTypeName = baseType.getSymbol().getEscapedName();
```

Reply:
Ok got it

Reply:
Hopefully this resolves it